jack_bot.py

```diff
@@ -232,27 +174,6 @@ def set_game_step(self):
             self.prepare_expansions()
             self.split_workers()
 
-        if self.ground_enemies:
```

Comment:
here too is missing on the new one

Reply:
please review https://github.com/Matuiss2/Greedy_bot/pull/148 first, once minor-fixes is merged, this PR will be less of code and only contains the changes to dodging